broker-daemon/proto/broker.proto

```diff
@@ -360,16 +360,43 @@ message Order {
     CANCELLED = 6;
   }
 
+  message Date {
+    // The date the order has been created on the Relayer.
+    string date_created = 1;
+    // The date the order has been placed and communicated to other marker participants.
+    string date_placed = 2;
+    // The date the swap is being executed on the Payment Channel Networks
+    string date_executing = 3;
+    // The date the swap preimage has been returned to the Relayer.
+    string date_completed = 4;
+    // The date the order has encountered a failure.
+    string date_rejected = 5;
+    // The date the order has been cancelled by the user.
+    string date_cancelled = 6;
+  }
+
   // Unique ID assigned by the Relayer for this order.
   string order_id = 1;
   // Status of the order on the Relayer.
-  OrderStatus order_status = 2;
+  OrderStatus status = 2;
   // Amount of the order expressed in a decimal string of common units for a currency (e.g. bitcoins for BTC).
   string amount = 3;
   // Price at which the order was placed, expressed as a decimal string ratio between the common units of the currencies (e.g. litecoins to bitcoins, not litoshis to satoshis). All orders on the SparkSwap Relayer have prices.
   string price = 4;
   // Any error messages related to the status of a REJECTED or FAILED order
-  string order_error = 6;
+  string error = 6;
+  // The unique ID for the block order assigned by the Broker that this order belongs to.
+  string block_order_id = 7;
+  // The base currency for the order
+  string base_symbol = 8;
+  // The counter currency for the order
+  string counter_symbol = 9;
+  // The side of the market that the order is taking.
+  string side = 10;
+  // Dates that the order has reached each status
+  Date dates = 11;
+  // Amount of the order filled, expressed in a decimal string of common units for a currency (e.g. bitcoins for BTC). This can be any amount greater than zero and less than or equal to the amount in the order.
```

Comment:
I would note that this will be an empty string if the order has not been filled.